api/spec/requests/v1/services_controller_spec.rb

```diff
@@ -274,6 +274,21 @@
             expect(parse_response_json(response)).to include(expected_attributes)
           end
         end
+
+        context 'when an admin user updates the ending of a service' do
```

Comment:
Maybe it is worth protecting it agains civil servant after confirmation date?

Reply:
Implemented this